mail-mta/protonmail-bridge-bin/protonmail-bridge-bin-1.0.6.ebuild

```diff
@@ -0,0 +1,188 @@
+	rm "plugins/sensorgestures/libqtsensorgestures_plugin.so"
+	rm "plugins/sensorgestures/libqtsensorgestures_shakeplugin.so"
+	rm "plugins/sensors/libqtsensors_generic.so"
+	rm "plugins/sensors/libqtsensors_iio-sensor-proxy.so"
+	rm "plugins/sensors/libqtsensors_linuxsys.so"
+	rm "plugins/sqldrivers/libqsqlite.so"
+	rm "plugins/sqldrivers/libqsqlmysql.so"
+	rm "plugins/sqldrivers/libqsqlpsql.so"
+	rm "plugins/xcbglintegrations/libqxcb-egl-integration.so"
+	rm "qml/Qt/labs/calendar/libqtlabscalendarplugin.so"
+	rm "qml/Qt/labs/folderlistmodel/libqmlfolderlistmodelplugin.so"
+	rm "qml/Qt/labs/platform/libqtlabsplatformplugin.so"
+	rm "qml/Qt/labs/settings/libqmlsettingsplugin.so"
+	rm "qml/Qt/labs/sharedimage/libsharedimageplugin.so"
+	rm "qml/Qt3D/Animation/libquick3danimationplugin.so"
+	rm "qml/Qt3D/Core/libquick3dcoreplugin.so"
+	rm "qml/Qt3D/Extras/libquick3dextrasplugin.so"
+	rm "qml/Qt3D/Input/libquick3dinputplugin.so"
+	rm "qml/Qt3D/Logic/libquick3dlogicplugin.so"
+	rm "qml/Qt3D/Render/libquick3drenderplugin.so"
+	rm "qml/QtBluetooth/libdeclarative_bluetooth.so"
+	rm "qml/QtCanvas3D/libqtcanvas3d.so"
+	rm "qml/QtCharts/libqtchartsqml2.so"
+	rm "qml/QtDataVisualization/libdatavisualizationqml2.so"
+	rm "qml/QtGamepad/libdeclarative_gamepad.so"
+	rm "qml/QtLocation/libdeclarative_location.so"
+	rm "qml/QtMultimedia/libdeclarative_multimedia.so"
+	rm "qml/QtNfc/libdeclarative_nfc.so"
+	rm "qml/QtPositioning/libdeclarative_positioning.so"
+	rm "qml/QtPurchasing/libdeclarative_purchasing.so"
+	rm "qml/QtQml/Models.2/libmodelsplugin.so"
+	rm "qml/QtQml/StateMachine/libqtqmlstatemachine.so"
+	rm "qml/QtQuick/Controls.2/Material/libqtquickcontrols2materialstyleplugin.so"
+	rm "qml/QtQuick/Controls.2/Universal/libqtquickcontrols2universalstyleplugin.so"
+	rm "qml/QtQuick/Controls/Styles/Flat/libqtquickextrasflatplugin.so"
+	rm "qml/QtQuick/Controls/libqtquickcontrolsplugin.so"
+	rm "qml/QtQuick/Dialogs/Private/libdialogsprivateplugin.so"
+	rm "qml/QtQuick/Dialogs/libdialogplugin.so"
+	rm "qml/QtQuick/Extras/libqtquickextrasplugin.so"
+	rm "qml/QtQuick/LocalStorage/libqmllocalstorageplugin.so"
+	rm "qml/QtQuick/Particles.2/libparticlesplugin.so"
+	rm "qml/QtQuick/PrivateWidgets/libwidgetsplugin.so"
+	rm "qml/QtQuick/Scene2D/libqtquickscene2dplugin.so"
+	rm "qml/QtQuick/Scene3D/libqtquickscene3dplugin.so"
+	rm "qml/QtQuick/VirtualKeyboard/Styles/libqtvirtualkeyboardstylesplugin.so"
+	rm "qml/QtQuick/XmlListModel/libqmlxmllistmodelplugin.so"
+	rm "qml/QtScxml/libdeclarative_scxml.so"
+	rm "qml/QtSensors/libdeclarative_sensors.so"
+	rm "qml/QtTest/libqmltestplugin.so"
+	rm "qml/QtWebChannel/libdeclarative_webchannel.so"
+	rm "qml/QtWebEngine/libqtwebengineplugin.so"
+	rm "qml/QtWebSockets/libdeclarative_qmlwebsockets.so"
+	rm "qml/QtWebView/libdeclarative_webview.so"
+
+	default
+}
+
+src_install() {
+	# Using doins -r would strip executable bits from all binaries
+	cp -pPR "${S}"/usr "${D}"/ || die "Failed to copy files"
```

Comment:
`${S}` is the current directory here, so no need to repeat it.

Reply:
removed